### PR TITLE
fix: UI layouts for topbar and sidebar

### PR DESCRIPTION
**Pull request by ayoubbajjou**
UI layouts for topbar and sidebar
_ Steer Campaign_ Steer Campaign (3)_ Steer Campaign (4)_ Steer Campaign (5)

### CODE REVIEW

File: config/jetstream.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
     'features' => [
         // Features::termsAndPrivacyPolicy(),
-        // Features::profilePhotos(),
+        Features::profilePhotos(),
```
Comment (mabumusa1) [inline config/jetstream.php:46]: We do not want to use `profilePhotos` feature, we do not want to upload profile pictures, let's just use this feature of Metronic. to show the first letter from the first and last name.

https://preview.keenthemes.com/metronic8/demo1/documentation/base/symbol.html#label

File: routes/web.php [hunk 2]
```diff
@@ -20,3 +20,7 @@
 Route::middleware(['auth:sanctum', 'verified'])->get('/dashboard', function () {
     return view('dashboard');
 })->name('dashboard');
+
```
Comment (mabumusa1) [inline routes/web.php:23]: Let's use the routes in the `dev` branch, there is conflict on this file

File: config/jetstream.php (hunk 1 above)
Comment (ayoubbajjou) [reply config/jetstream.php:46]: yeah, i can do something like in the switch account and remove this feature.

File: routes/web.php (hunk 2 above)
Comment (ayoubbajjou) [reply routes/web.php:23]: should i remove this route ?

File: config/jetstream.php (hunk 1 above)
Comment (mabumusa1) [reply config/jetstream.php:46]: @ayoubbajjou yes, please do that. 

We are trying to keep the system as compact as possible

File: routes/web.php (hunk 2 above)
Comment (ayoubbajjou) [reply routes/web.php:23]: i think i should just fix this conflict and leave it there, what do you think ?

File: config/jetstream.php (hunk 1 above)
Comment (ayoubbajjou) [reply config/jetstream.php:46]: Ok, sounds good.

File: routes/web.php (hunk 2 above)
Comment (mabumusa1) [reply routes/web.php:23]: These are the final routes https://github.com/mabumusa1/saas/blob/dev/routes/web.php, I think if you rebase we will not have conflict

**Pull request closed without merging**